admin/cincoctrl/cincoctrl/findingaids/validators.py

```diff
@@ -0,0 +1,16 @@
+import xml.etree.ElementTree as ET
+
+from django.core.exceptions import ValidationError
+
+def validate_ead(file):
```

Comment:
I feel like the EAD validation is going to get more and more complex as we go on. Probably worth adding a test now so it is easy to new validation logic as we go forward and don't accidentally backslide.